new.rst

```diff
@@ -0,0 +1,97 @@
+.. _whats_new:
+
+###############################
+What's New in {Singularity} 4.0
+###############################
+
+This section highlights important changes in {Singularity} 4.0 that are of note
+to users. See also the "What's New" section in the Admin Guide for
+administrator-facing changes.
+
+********
+OCI-mode
+********
+
+Singularity 4 introduces :ref:`OCI-mode <oci_mode>` as a fully supported
+feature. It is enabled by using the ``--oci`` flag with the run / shell / exec /
+pull commands, or when ``oci mode = yes`` is set in ``singularity.conf``.
+
+In OCI-mode:
+
+- Container images from OCI sources will be pull-ed to an :ref:`OCI-SIF
+  <oci_sif>` file. An OCI-SIF file encapsulates the OCI image configuration and
+  squashed filesystem using an OCI, rather than Singularity specific, structure.
+- The run / shell / exec commands use a low-level OCI runtime (crun/runc) for
+  container execution.
+- Default operation is :ref:`compatible with other OCI tools <oci_compat>`,
+  similar to ``--compat`` in Singularity's non-OCI native mode.
+- OCI-mode supports running existing Singularity non-OCI-SIF images, and can be
```

Comment:
"non-OCI-SIF" is ambiguous, I think, between "[non-OCI]-SIF" (=SIFs that are non-OCI) and "non-[OCI-SIF]" (=images that are not OCI-SIF) - the latter being what (I think) was intended...? In any event, I suggest rephrasing in a way that's not ambiguous, perhaps using the parenthesized text above.